source/extensions/filters/http/composite/filter.cc

```diff
@@ -105,7 +105,7 @@ void Filter::onMatchCallback(const Matcher::Action& action) {
                   wrapper.errors_, [](const auto& status) { return status.ToString(); }));
     return;
   }
-  std::string actionName = composite_action.actionName();
+  const std::string& actionName = composite_action.actionName();
```

Comment:
nit: `action_name`?

Reply:
i can change it - not sure why it wasnt linted (not sure i fully follow the rules tbh)